x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -15238,8 +15238,8 @@
     "xpack.securitySolution.auditd.errorFromDescription": "エラー：",
     "xpack.securitySolution.auditd.executedDescription": "実行",
     "xpack.securitySolution.auditd.executionOfForbiddenProgramDescription": "禁止されたプログラムの実行",
-    "xpack.securitySolution.auditd.failedLoginTooManyTimesDescription": "ログイン回数超過によりログインに失敗",
-    "xpack.securitySolution.auditd.inDescription": "in",
+    "xpack.securitySolution.auditd.failedxLoginTooManyTimesDescription": "ログイン回数超過によりログインに失敗",
+    "xpack.securitySolution.auditd.inDescription": "次に含まれる：",
```

Comment:
Did you translate this manually? I think we should probably leave it how it exists on `master`.

Reply:
I didn't translate this manually. I think when I tried to remove `xpack.securitySolution.auditd.failedxLoginTooManyTimesDescription` initially and did some cut and paste I must have replaced it accidentally.